core/src/main/scala/scalaz/Order.scala

```diff
@@ -1,6 +1,8 @@
 package scalaz
 
 ////
+import scalaz.Ordering.{GT, LT}
```

Comment:
here too, it's idiomatic to use the relative path instead of adding an import at the top.